assets/src/edit-story/components/canvas/page.js

```diff
@@ -43,6 +43,9 @@ function Page() {
 	}, [ setBackgroundClickHandler, clearSelection ] );
 
 	const handleSelectElement = useCallback( ( elId, evt ) => {
+		if ( editingElement && elId !== editingElement ) {
```

Comment:
I suppose later separate handling is needed for the image element to keep the selection? (Based on #3994)

Reply:
Yeah. This is strictly to fix a problem. Not a feature development.